Tribler/Core/Upgrade/upgrade.py

```diff
@@ -39,6 +40,11 @@ def run(self):
             if has_to_upgrade and not failed:
                 self.notify_starting()
                 self.upgrade_database_to_current_version()
+
+                # Convert old (pre 6.3 Tribler) pickle files to the newer .state format
```

Comment:
Oh, I see the version here. Was 6.2 running an older than V17 database? that's where we draw the lines re. DB upgrades. So if this is older, we could just drop it.